tests/test_timeseries.py

```diff
@@ -0,0 +1,53 @@
+# TODO: Temporary mock setup while waiting for a better way to do integration tests
+from datetime import datetime
+from unittest.mock import patch
+
+import pytest
+
+from cognite.timeseries import get_datapoints
+
+
+@patch('requests.get')
+@pytest.fixture(scope='module', params=[
+    {'start': 0, 'end': 1},
+    {'start': '1w-ago', 'end': '1d-ago'},
+    {'start': datetime(2018, 3, 26), 'end': datetime(2018, 3, 27)}])
+def get_datapoints_response_obj(mock_get, request):
+    mock_get.return_value.status_code = 200
```

Comment:
Same as for the previous one. I would prefer a more proper mock object such as MockReturnValue in conftest